client/src/app.jsx

```diff
@@ -0,0 +1,147 @@
+              <p>2. Just ask Alexa for music, weather, news, and more.</p>
+            </div>
+            <div>
+              <img className="feature-block-7-media" src={img.featuresBlock8[2]}></img>
+              <p>3. For simple setup with compatible smart home devices, say, “Alexa, discover my devices.”</p>
+            </div>
+          </div>
+        </div>
+
+        <div className="additional-features">
+          <div className="additional-features-header">
+            <h2>Every day is an earth day</h2>
+          </div>
+          <div className="additional-features-text">
+            <p>You don’t have to be an environmental scientist to make sustainable choices. We considered sustainability in the design of this device. Here’s how:</p>
+          </div>
+          <div className="additional-features-rows">
+            <div className="additional-features-row">
+              <img className="additional-features-icon" src={img.featuresBlock9[0]}></img>
+              <div className="additional-features-item">
+                <h4>Materials</h4>
+                  <p>50% post-consumer recycled plastics<br/>
+                  100% post-consumer recycled fabric<br/>
+                  100% recycled die-cast aluminum<br/>
+                  Device, cable and adapter included</p>
+              </div>
+              <img className="additional-features-icon" src={img.featuresBlock9[1]}></img>
+              <div className="additional-features-item">
+                <h4>Clean Energy</h4>
+                  <p>Amazon is building new wind and solar farms to produce clean energy equivalent to the electricity used by this device.</p>
+              </div>
+              <img className="additional-features-icon" src={img.featuresBlock9[2]}></img>
+              <div className="additional-features-item">
+                <h4>Low Power Mode</h4>
+                  <p>To reduce its energy consumption, this device will enter Low Power Mode when it is idle. Low Power Mode is not available for certain user configurations. To learn more, see Low Power Mode Restrictions.</p>
+              </div>
+            </div>
+            <div className="additional-features-row">
+              <img className="additional-features-icon" src={img.featuresBlock9[3]}></img>
+              <div className="additional-features-item">
+                <h4>Alexa</h4>
+                  <p>Track and manage the energy usage of your Alexa-connected devices with a new energy dashboard, coming soon to the Alexa app.</p>
+              </div>
+              <img className="additional-features-icon" src={img.featuresBlock9[4]}></img>
+              <div className="additional-features-item">
+                <h4>Device Packaging</h4>
+                  <p>99% of this device’s packaging is made of wood fiber-based materials from responsibly managed forests or recycled sources.</p>
+              </div>
```

Comment:
Looks like all the header and paragraphs are hardcoded. These content would change from product to product. Shouldn't these be driven from database similar to how you have image coming from Database ?

Reply:
Yes! This is static version of page, and data is hard coded for now. This will be refactored when all API endpoints are available for Blitva!